drivers/CoreFs/include/LKCoreFatFs.h

```diff
@@ -0,0 +1,32 @@
+// Leka - LekaOS
+// Copyright 2021 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef _LEKA_OS_LIB_FATFS_H_
+#define _LEKA_OS_LIB_FATFS_H_
+
+#include "LKCoreFatFsBase.h"
+
+namespace leka {
+
+class LKCoreFatFs : public LKCoreFatFsBase
```

Comment:
As reminder

```suggestion
class CoreFatFs : public CoreFatFsBase
```

Ainsi que le nom du fichier